src/requestHandlers/panel.ts

```diff
@@ -0,0 +1,120 @@
+import { Request, Response } from 'express';
+import * as panelService from '../services/panelService';
+import { genericErrorResponse } from './helpers';
+
+
+interface ResponseObject {
+    success: boolean,
+    body?: any
+    message?: string,
+    error?: string,
+    status?: number,
+}
+
+/**
+ * Creates a panel
+ * @param image //image path
+ * @param index //panel index
+ * @param panel_set_id //id of the panel set the panel is a part of
+ * @returns {ResponseObject}
+ */
+const _createPanelController = async (image: string, index: number, panel_set_id: number) => {
+    try {
+        const response = await panelService.createPanel({
+            image:        image,
+            index:        index,
+            panel_set_id: panel_set_id,
+        });
+
+        return {
+            success: true,
+            body:    response
+        } as ResponseObject;
```

Comment:
Any reason not to just return `response` here? Let a thrown error mean unsuccess.

Reply:
I was following what Arthur had already set up. The reason was to return a specific error code like 400 for a fail rather than always 200. But I suppose the response itself has the error shown in it anyway.

Reply:
I updated to only return the response 
```
const _getPanelController = async (id:number) => {
    try {
        const response = await panelService.getPanel(id);

        return response;
    }
    catch (err) {
        return genericErrorResponse(err as Error);
    }
};

// the actual request for getting a panel
const getPanel = async (req: Request, res: Response): Promise<Response> => {
    const response = await _getPanelController(req.body.id,);

    return res.status(200).json(response);
};